# request for picking a block

# rqst: which arm it is and where it is
string arm
geometry_msgs/Pose eef_pose

---

# rply: which block it is and where it is

string block_name
string block_color
geometry_msgs/Pose block_pose
